docs/getting-started/recurring-tasks.md

```diff
@@ -50,7 +50,9 @@ The next Sunday after 25 April 2021 is on 2 May.
 Important
 {: .label .label-yellow }
 
-A recurring task should have a due date. The due date and the recurrence rule must appear after the task's description.
+A recurring task should have at least one of start, scheduled and due dates. The date(s) and the recurrence rule must appear after the task's description.
+
+(Technically, you _can_ add a recurrence rule to a task without any dates, but it is not clear that the behaviour is very useful.)
```

Comment:
> but it is not clear that the behaviour is very useful

What is meant by this? - maybe an example would help me

Reply:
It was an attempt to avoid going in to detail, by glossing over it!

Here is a recurring task without any dates, that I completed several times - in the real world, the done dates would be different, of course:

```
- [ ] Do stuff 🔁 every day
- [x] Do stuff 🔁 every day ✅ 2023-02-13
- [x] Do stuff 🔁 every day ✅ 2023-02-13
```

They are not searchable by date, and I think most people find their tasks by searching for dates.

I'm a believer in docs being honest, but I was thinking that going in to detail about how the above works will be more of a distraction to people using recurrence for the normal case of with dates...

Reply:
thanks for the detailed response!